src/Controller/EnMarche/CitizenProjectMediaGeneratorController.php

```diff
@@ -0,0 +1,70 @@
+<?php
+
+namespace AppBundle\Controller\EnMarche;
+
+use AppBundle\Form\CitizenProjectImageType;
+use AppBundle\Form\CitizenProjectTractType;
+use Knp\Bundle\SnappyBundle\Snappy\Response\PdfResponse;
+use Sensio\Bundle\FrameworkExtraBundle\Configuration\Method;
+use Sensio\Bundle\FrameworkExtraBundle\Configuration\Route;
+use Symfony\Bundle\FrameworkBundle\Controller\Controller;
+use Symfony\Component\HttpFoundation\Request;
+use Symfony\Component\HttpFoundation\Response;
+
+/**
+ * @Route("/projets-citoyens/media-generateur")
+ */
+class CitizenProjectMediaGeneratorController extends Controller
+{
+    /**
+     * @Route("/images", name="app_citizen_project_image_generator")
+     * @Method({"GET", "POST"})
+     */
+    public function generateImageAction(Request $request): Response
+    {
+        $form = $this
+            ->createForm(CitizenProjectImageType::class)
+            ->handleRequest($request);
+
+        $coverImage = null;
+
+        if ($form->isSubmitted() && $form->isValid()) {
+            $coverImage = $this->get('app.citizen_project.cover_generator')->generate($form->getData());
+        }
+
+        return $this->render(
+            'citizen_project/media_generator_image_form.html.twig',
+            [
+                'form' => $form->createView(),
+                'previewCoverImage' => $coverImage ? $coverImage->getContentAsDataUrl() : null,
+            ]
+        );
+    }
+
+    /**
+     * @Route("/tracts", name="app_citizen_project_tract_generator")
+     * @Method({"GET", "POST"})
+     */
+    public function generateTractAction(Request $request): Response
+    {
+        $form = $this
+            ->createForm(CitizenProjectTractType::class)
+            ->handleRequest($request);
```

Comment:
CS
```
    ->handleRequest($request)
;
```

Reply:
is neither PSR, nor Symfony php-cs-fixer rule

Reply:
is the project rule then 💃

Reply:
[done](https://github.com/EnMarche/en-marche.fr/pull/2167/commits/2c8961cb0faaa0fb39faf62750cd7127b4d4a281#diff-8b6c0c683d533f32b31ce28cee1bd3e6R17) ;)